app/controllers/patients_controller.rb

```diff
@@ -1,2 +1,5 @@
 class PatientsController < ApplicationController
+  def index
+    @patients = current_user.patients.uniq
```

Comment:
I think in the demo we will not have the concept of shift and patient of the shift so getting all patients of the nurse should be fine